src/components/status-bar.component.tsx

```diff
@@ -8,20 +8,13 @@ import { styled, StyledComponentProps } from '@ui-kitten/components';
 
 export type StatusBarProps = RNStatusBarProps & StyledComponentProps;
 
+@styled('StatusBar')
 class StatusBarComponent extends React.Component<StatusBarProps> {
-
-  static styledComponentName: string = 'StatusBar';
-
   public render(): React.ReactElement<ViewProps> {
-    const { themedStyle, ...statusBarProps } = this.props;
+    const { ...statusBarProps } = this.props;
 
-    return (
-      <RNStatusBar
-        {...themedStyle}
-        {...statusBarProps}
-      />
-    );
+    return <RNStatusBar {...statusBarProps} />;
   }
 }
```

Comment:
Make sure to not skip handling `eva` property when [working with `styled` components](https://akveo.github.io/react-native-ui-kitten/docs/components/using-mapping/overview#styled).
The previous code had no effect to RNStatusBar component because eva mapping was applied incorrectly.

```suggestion
public render(): React.ReactElement< ViewProps > {
    const { eva, ...statusBarProps } = this.props;

    return (
      <RNStatusBar
        {...eva?.style}
        {...statusBarProps}
      />
    );
  }
}
```